configuration/configuring-date-and-time/en.md

```diff
@@ -3,8 +3,50 @@ title = "Configuring Date and Time"
 +++
 # Configuring Date and Time
 
-You can configure the date and time of your system by going to the Budgie Menu, opening the Settings app and then going to the Date & Time section.
+With Solus it is easy to change the date and time of your system and how it is displayed.
+
+## Budgie
+
+In Budgie, the easiest way to access the date and time settings is by left clicking on the time in the top panel and clicking on "Time and date settings".  
+
+{{< altimg "budgie-time-and-date-right-click.jpg" "help-center/configuration/configuring-date-and-time/" >}}
+
+Alternatively, you can open the Budgie Menu and then click on "Date & Time" in the System Settings section.
+
+You will be presented with the following window. This window provides you with the ability to enable / disable automatic date & time, automatic time zone changing, and the time format.  You will need to click on "Unlock" and input your user password before changes can be made.
+
+{{< altimg "configuring-date-and-time.jpg" "help-center/configuration/configuring-date-and-time/" >}}
+
+If you wish to change how the time is displayed in the top panel, left click on the time and click "Preferences".  You can now show the date, seconds or use 24 hour time.
```

Comment:
I'm going to add this post-merge:

![image](https://cloud.githubusercontent.com/assets/156574/26783054/f076a896-49fe-11e7-869f-31b44f1bf771.png)